FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt

```diff
@@ -33,6 +33,7 @@ const val V2SEARCHBAR = "V2 SearchBar"
 const val V2SEGMENTED_CONTROL = "V2 SegmentedControl"
 const val V2SHIMMER = "V2 Shimmer"
 const val V2SNACKBAR = "V2 Snackbar"
+const val V2SUGGESTIONPROMPT = "V2 Suggestion Prompt"
```

Comment:
Lets add a "_" in suggestion prompt constant

Reply:
Fixed